downstream/modules/hub/proc-adding-an-execution-environment.adoc

```diff
@@ -1,26 +1,26 @@
 
 [id="adding-an-execution-environment"]
 
-= Adding an execution environment
+= Adding an {ExecEnvShort}
 
 .Procedure
-. Navigate to menu:Execution Environments[].
+. Navigate to menu:Execution Environments[Add execution environment].
```

Comment:
This should be:
Navigate to menu:Execution Environments[Execution Environments].
Click btn:[Add execution environment].